docs/source/history.rst

```diff
@@ -17,7 +17,7 @@ All execution records are accessible by the ``st2 execution *`` command family.
 
 .. code-block:: bash
 
-    # to see last 5 executions
+    # to see last 5 liveactions
```

Comment:
put a space between live actions in UI/CLI output?  i.e. "to see last 5 live actions"